hack/install-assets.sh

```diff
@@ -38,6 +38,10 @@ if ! which grunt > /dev/null 2>&1 ; then
 fi
 
 pushd ${OS_ROOT}/assets > /dev/null
+  # Check whether we need to force an update of the npm packages
+  if git diff master HEAD | grep 'assets/package.json' > /dev/null 2>&1 ; then
+    cmd "rm -rf node_modules"
```

Comment:
@spadgett so the -force option just forces it to skip the npm cache, which didnt help this.  how do you feel about this change instead